pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -88,13 +90,24 @@ class WorldSessionActor extends Actor with MDCContextAware {
   var flying : Boolean = false
   var speed : Float = 1.0f
   var spectator : Boolean = false
-  var admin : Boolean = false
+  var admin : Boolean = true
+  var noSpawnPointHere : Boolean = false
   var usingMedicalTerminal : Option[PlanetSideGUID] = None
   var controlled : Option[Int] = None
   //keep track of avatar's ServerVehicleOverride state
   var traveler : Traveler = null
   var deadState : DeadState.Value = DeadState.Dead
+  var whenUsedLastAAMAX : Long = 0
+  var whenUsedLastAIMAX : Long = 0
+  var whenUsedLastAVMAX : Long = 0
+  var whenUsedLastMAX : Array[Long] = Array.fill[Long](4)(0L)
+  var whenUsedLastMAXName : Array[String] = Array.fill[String](4)("")
+  var whenUsedLastItem : Array[Long] = Array.fill[Long](1020)(0L)
+  var whenUsedLastItemName : Array[String] = Array.fill[String](1020)("")
   var whenUsedLastKit : Long = 0
+  var whenUsedLastSMKit : Long = 0
+  var whenUsedLastSAKit : Long = 0
+  var whenUsedLastSSKit : Long = 0
```

Comment:
We definitely need to improve this on a followup patch. Lots of duplication

Reply:
Definitely. I didn't want to make any changes to soun's code until we've got the repos more closely aligned

Reply:
@Mazo you can play, I'm a noob ;)